Files/Helpers/FileThumbnailHelper.cs

```diff
@@ -8,19 +8,18 @@ namespace Files.Helpers
 {
     public static class FileThumbnailHelper
     {
-        private static AppServiceConnection AppServiceConnection { get; set; }
         public static async Task<(byte[] IconData, byte[] OverlayData, bool IsCustom)> LoadIconOverlayAsync(string filePath, uint thumbnailSize)
         {
-            AppServiceConnection ??= await AppServiceConnectionHelper.Instance;
-            if (AppServiceConnection != null)
+            var connection = await AppServiceConnectionHelper.Instance;
+            if (connection != null)
             {
                 var value = new ValueSet
                 {
                     { "Arguments", "GetIconOverlay" },
                     { "filePath", filePath },
                     { "thumbnailSize", (int)thumbnailSize }
                 };
-                var response = await AppServiceConnection.SendMessageAsync(value);
+                var response = await connection.SendMessageAsync(value);
```

Comment:
This PR makes me notice that the above should be changed to `connection.SendMessageSafeAsync()` which handles exceptions and is used everywhere else. Probably best done in a new PR.

Reply:
@gave92 Got it, I'll make it once this is merged to avoid conflicts.